dist-git-to-src-git/README.md

```diff
@@ -0,0 +1,78 @@
+# Converting dist-git to source-git
+
+This should be similar to [How to source-git?], except that it isn't :)
+
+The main difference is that [How to source-git?] starts with the upstream
+history and applies the patches from dist-git.
+
+In this case though, the starting point is dist-git itself, so a few things
+need to be done differently.
+
+Here are the steps:
+
+*Note:* things bellow were done on git.centos.org/rpms/rpm.
+
+1. Fetch the sources from the lookaside cache.
+
+   This will create some `tar.*` under `SOURCES`.
+
+2. Unpack the `tar` to `src/rpm`.
+
+3. Now applying the patches (from the `SOURCES` directory) can begin. In order
+   to decide which patches to apply, look in the spec-file. Use
+   [rebase-helper] for this.
```

Comment:
why not just do centpg/fedpkg `prep`?

Reply:
b/c I wasn't aware of it :)

Reply:
I can see that `prep` applies also all the patches after it unpacks the archive

I wonder if we need to apply patches in a dedicated step

Reply:
Not necessarily, but the patches that are applied need to be removed from the spec file in order to keep things in a working state.

I'll have a look at what exactly `fedpkg prep` does.

In general I agree that it would be better to use existing tooling than to reinvent the wheel.

Reply:
fedpkg prep is just a wrapper around `rpmbuild -bp`

Reply:
you can also look at what packit srpm does, same concept, different rpmbuild command

Reply:
I've looked into all the above. 

While all the methods above would unpack the source and apply the patches, in the case of dist-git to src-git conversion, those patches also need to be removed from the spec file. 

Hooking this into the approaches mentioned above seems to me to be at least as complicated (if not more) than the current implementation (we would still need to get the list of patches applied and comment them out from the SPEC-file).

So if you agree, I would stick to the current implementation, at least, I don't see any solid benefit to change it now. WDYT?